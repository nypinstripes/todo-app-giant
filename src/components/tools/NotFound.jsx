import React from 'react';
import SvgSymbol from './SvgSymbol';

const NotFound = props => (
  <div className="not-found">
    <h1>Zoink!</h1>
    <div className="not-found-icon">
      <div className="icon-gear">
        <SvgSymbol symbolId="#icon-gear-large" />
      </div>
      <div className="icon-gear">
        <SvgSymbol symbolId="#icon-gear-small" />
      </div>
    </div>
    <p>Didn&apos;t find that one.</p>
  </div>
);

export default NotFound;
