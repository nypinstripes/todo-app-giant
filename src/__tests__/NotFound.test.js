import React from 'react';
import { create } from 'react-test-renderer';
import NotFound from '../components/tools/NotFound';

test('snapshot', () => {
  const notFoundComponent = create(<NotFound />);

  expect(notFoundComponent.toJSON()).toMatchSnapshot();
});
