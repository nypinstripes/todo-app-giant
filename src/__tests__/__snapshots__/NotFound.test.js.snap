// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="not-found"
>
  <h1>
    Zoink!
  </h1>
  <div
    className="not-found-icon"
  >
    <div
      className="icon-gear"
    >
      <svg
        height="0"
        width="0"
      >
        <use
          x="0"
          xlinkHref="#icon-gear-large"
          y="0"
        />
      </svg>
    </div>
    <div
      className="icon-gear"
    >
      <svg
        height="0"
        width="0"
      >
        <use
          x="0"
          xlinkHref="#icon-gear-small"
          y="0"
        />
      </svg>
    </div>
  </div>
  <p>
    Didn't find that one.
  </p>
</div>
`;
